components/SpotlightBanner/__tests__/SpotlightBanner.spec.tsx

```diff
@@ -18,7 +18,7 @@ describe('SpotlightBanner', () => {
   test('renders only SpotlightBannerAside & SpotlightBannerMain', () => {
     const renderer = TestRenderer.create(
       <SpotlightBanner>
-        <SpotlightBannerAside />
+        <SpotlightBannerAside classname={'spotlight__comparison'} />
```

Comment:
Ahh, I see why u removed the aside class ... nope, a different component should have the `spotlight_comparison` class... they clearly server a different purpose and don't belong in the same component. This component is already created in the `feature/compare-indicators` branch.